docs/sources/alerting/set-up/performance-limitations/index.md

```diff
@@ -65,6 +65,10 @@ This can be prevented by writing to the database periodically. For this the feat
 to be enabled. By default, it saves the states every 5 minutes to the database and on each shutdown. The periodic interval
 can also be configured using the `state_periodic_save_interval` configuration flag.
 
+When periodic saving is enabled, each `state_periodic_save_interval` Grafana deletes all existing alert instances from the
```

Comment:
not sure what the first sentence , each state periodic save interval Grafana means?

Reply:
I updated the doc, could you have another look, please?